### PR TITLE
Creates new page detailing Bedrock connector setup

### LINKED ISSUE #5113: [Request]Connect Elastic AI Assistant to Amazon Bedrock
### Description

Giving step by step instructions on how to connect Elastic AI Assistant with AWS Bedrock. 

### Background & resources

Document Link: https://docs.google.com/document/d/1ebrw6gHiGxbblopYZybV1jVu1ulVe_3hLDb6QtTzY4s/edit?usp=sharing

Video Files Link: https://drive.google.com/drive/folders/1Xs9TiABjSPddnAZzhpqLtlOMvQroyLxQ?usp=drive_link

### Which documentation set does this change impact?

ESS and serverless

### ESS release

N/A

### Serverless release

N/A

### Feature differences

This clarifies step by step assistance to make it easier for users to connect to AWS Bedrock.

### API docs impact

What: This clarifies step by step assistance to make it easier for users to connect to AWS Bedrock.

Why: User have a hard time adopting AI Assistant, we are trying to make the steps as easy as possible to follow.

### Prerequisites, privileges, feature flags

None

### PR DESCRIPTION
**Pull request by benironside**
Fixes #5113 by adding a page with step by step details for how to set up an Amazon Bedrock connector.

Preview:[Set up an Amazon Bedrock connector](https://security-docs_bk_5148.docs-preview.app.elstc.co/guide/en/security/master/assistant-connect-to-bedrock.html)

### CODE REVIEW

**Review by joepeeples — COMMENTED** [review]
Added a few small line edits. Also, it'd help to give users a bit more context and explanation of what they're doing overall, and in each subsection. I think these steps are clear enough for users to follow, but we could tell them more about what they're doing, and why. 

If Amazon changes their UI in the future (and of course they will), then our users will be better able to adjust if we equip them with a good understanding of broadly what they need to do. If they're just blindly following steps, it'll be harder for them to adjust.

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+. Select **Third-party service**, check the box under **Confirmation**, click **Next**, then click **Create access key**.
+. Click **Download .csv file** to download the key. Store it securely.
+
+[discrete]
+=== Enable model access
+
+. Search the AWS console for Amazon Bedrock. 
+. From the Amazon Bedrock page, click **Get started**.
+. Select **Model access** from the left navigation menu, then click **Manage model access**.
+. Check the boxes for **Claude** and **Claude Instant**, depending which model or models you plan to use.
```
Comment (Charelzard) [inline docs/assistant/connect-to-bedrock.asciidoc:64]: Check the boxes for **Claude** and/or **Claude Instant**, depending which model or models you plan to use.

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
+
+Your model is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.
+
+IMPORTANT: If you're using provisioned throughput, your ARN becomes the model ID, and the connector settings **URL** value must be https://www.urlencoder.org/[encoded] to work.
```
Comment (Charelzard) [inline docs/assistant/connect-to-bedrock.asciidoc:82]: IMPORTANT: If you're using [provisioned throughput](https://docs.aws.amazon.com/bedrock/latest/userguide/prov-throughput.html), your ARN becomes the model ID, and the connector settings **URL** value must be [encoded](https://www.urlencoder.org/) to work.

**Review by Charelzard — COMMENTED** [review]
Other than the comments I added, everything looks good. Please be sure to include the videos in each respective section. I like how you went the route to add the connectors vs doing it in the ai assistant to give a more standardized approach. We can use this in the other docs going forward.

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+. Log in to {kib}. 
+. Go to **Stack Management → Connectors → Create connector → Amazon Bedrock**. 
+. Name your connector. 
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
+
```
Comment (benironside) [inline docs/assistant/connect-to-bedrock.asciidoc:158]:
```suggestion
+
```

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+. Name your connector. 
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
++
+Your LLM connector is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.
+
```
Comment (benironside) [inline docs/assistant/connect-to-bedrock.asciidoc:160]:
```suggestion
+
```

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+. Name your connector. 
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
++
+Your LLM connector is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.
++
```
Comment (benironside) [inline docs/assistant/connect-to-bedrock.asciidoc:160]:
```suggestion

```

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+. Go to **Stack Management → Connectors → Create connector → Amazon Bedrock**. 
+. Name your connector. 
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
++
+Your LLM connector is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.
```
Comment (benironside) [inline docs/assistant/connect-to-bedrock.asciidoc:159]:
```suggestion
Your LLM connector is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.

```

File: docs/assistant/connect-to-bedrock.asciidoc [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+IMPORTANT: If you're using https://docs.aws.amazon.com/bedrock/latest/userguide/prov-throughput.html[provisioned throughput], your ARN becomes the model ID, and the connector settings **URL** value must be https://www.urlencoder.org/[encoded] to work. 
++
```
Comment (benironside) [inline docs/assistant/connect-to-bedrock.asciidoc:162]:
```suggestion
```

**Review by joepeeples — APPROVED** [review]
LGTM! The added bits nicely guide the reader through this multi-step process.

**Pull request merged**